.github/ISSUE_TEMPLATE/tck_challenge.md

```diff
@@ -0,0 +1,21 @@
+---
+name: TCK Challenge
+about: Create a TCK Challenge
+title: ''
+labels: 'challenge'
+assignees: ''
+
+---
+
+**Challenged tests**
+List the challenged tests with the fully qualified classnames and then the test methods, e.g.
+ee.jakarta.tck.faces.test.javaee7.multiFieldValidation.Spec1IT#testFailingPreconditionsNotAfterAllInputComponents
+
+**TCK Version**
+Specify the version of the TCK, e.g. Jakarta Faces 4.0.x
```

Comment:
Let's have the specification name in the example text match the project. This will make it easier for anyone who opens an issue to not need to replace that part.

```suggestion
Specify the version of the TCK, e.g. Jakarta Connectors 2.1.x
```